application.go

```diff
@@ -34,14 +34,6 @@ type Applications struct {
 	Apps []Application `json:"apps"`
 }
 
-// IPAddressPerTask is used by IP-per-task functionality https://mesosphere.github.io/marathon/docs/ip-per-task.html
-type IPAddressPerTask struct {
```

Comment:
IP address per task seems to be removed. Is it because Marathon 1.5 does not support it anymore?

We should try hard not to break existing clients if somehow possible.